ArchiExtension/archiAnalyze/archiAnalyze.cabal

```diff
@@ -0,0 +1,23 @@
+name:                archiAnalyze
+version:             0.1.0.0
+synopsis:            Simple project template from stack
+description:         Please see README.md
+homepage:            https://github.com/githubuser/archiAnalyze#readme
```

Comment:
All files in ArchiExtension/archiAnalyze/* should be removed. They are obsolete.